examples/blog-multiple-authors/src/pages/$author.astro

```diff
@@ -45,7 +45,7 @@ export async function createCollection() {
 const author = authorData[collection.params.author];
 ---
 
-<html>
+<html dir="ltr" lang="en">
```

Comment:
Could you pull this out into a variable?
```astro
const lang = 'en';
---

<html lang={ lang ?? 'en' }>
```

Reply:
When should I pull into a variable instead of from content?

Reply:
Just pull from content if you can at all.

Reply:
I don't see the point here, it's always going to return `'en'`, no? I could see moving it out if this were a layout and you wanted to allow passing a long via a prop.

Reply:
yea I'm also fine with keeping it inline for now, I'd love for this to be more automatic in the future (ex: if you don't include it, we add it for you by default).

Do we support site-wide variable/config yet? Something like `Astro.site.lang` that would be defined in your config file would be great for this in the future.